app/controllers/score_controller.py

```diff
@@ -0,0 +1,44 @@
+import typing
+from flask import request, jsonify, Response
+from flask_restx import Resource
+from pydantic import TypeAdapter
+
+from app.controllers import ns_score
+from app.model.document import DocumentEdit
+from app.restx_dtos import document_edit_request, similarity_score_response
+from app.util.score_calculator import ScoreCalculator
+
+
+@ns_score.route("")
+class ScoreController(Resource):
```

Comment:
```suggestion
class F1ScoreController(Resource):
```

Spezifische Benennung